src/esp/gfx/GenericDrawable.cpp

```diff
@@ -113,6 +118,12 @@ void GenericDrawable::updateShaderLightingParameters(
 
 void GenericDrawable::draw(const Mn::Matrix4& transformationMatrix,
                            Mn::SceneGraph::Camera3D& camera) {
+  if (!isRendererCreated) {
+    return;
+  }
+  // Check that getMesh() doesn't error, i.e. that a mesh exists.
+  getMesh();
```

Comment:
No, you do not need to call this function here (I know you have sanity check within, but do not call it simply because you would like to do the santity check.)
You should use "CORRADE_ASSERT" directly.